routes.js

```diff
@@ -64,7 +64,7 @@ addRoute('/wer-wir-sind', WhoWeAre)
 addRoute('/wer-wir-sind/team-de', TeamDE)
 addRoute('/wer-wir-sind/team-sa', TeamSA)
 addRoute('/wie-sie-helfen', HowToSupport)
-addRoute('/wie-sie-helfen/pate-werden', BecomeSponsor)
+addRoute('/wie-sie-helfen/patenprogramm', BecomeSponsor)
```

Comment:
Umm no, it should be "masifunde-gemeinschaft" :)

Reply:
And what about the English version? I'd suggest "masifunde-community" for that.

Reply:
So it should be `/wie-sie-helfen/masifunde-gemeinschaf` ?

I don't think we have English versions for these URL's. Only the blog has some kind of slug system as long as I remember.

Reply:
Hehe, noooooo!

`/wie-sie-helfen/masifunde-gemeinschaft` 

:D

Reply:
And for English, OK, then we're good.

Reply:
Oki doki updated. No angry 🐑 ...